molecule/default/converge.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+
+  vars:
+
+  pre_tasks:
+
+
+  roles:
+    - role: ansible-role-nfs
```

Comment:
We're going to need to set up a pair of images to act as client + server in the molecule file, and then populate this section to configure both the client and server (See the role's readme, it documents the whole thing),  And then in the verify stage, we can test by creating a file on the client, and checking the server to see if it showed up.